ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -44,7 +75,7 @@ export default function EditGasDisplay({
           <div className="edit-gas-display__warning">
             <ActionableMessage
               className="actionable-message--warning"
-              message="Swaps are time sensitive. “Medium” is not reccomended."
+              message={warning}
```

Comment:
I'm confused... as it stands could there be a warning? there is no setter for this piece of state?

Reply:
This is a placeholder so that I could create the UI.  The actual data will be added in a follow up PR.